packages/cli/src/commands/list/index.ts

```diff
@@ -259,6 +276,12 @@ export default async function list(client: Client) {
               dep.target === 'production' ? 'Production' : 'Preview',
               chalk.gray(getDeploymentDuration(dep)),
               showUsername ? chalk.gray(dep.creator?.username) : '',
+              Object.keys(policy).length > 0 && dep.expiration
+                ? ToDate(dep.expiration)
+                : '',
+              Object.keys(policy).length > 0 && dep.proposedExpiration
+                ? ToDate(dep.proposedExpiration)
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (soft-blocking):** camelCase

```suggestion
                ? toDate(dep.proposedExpiration)
```